lib/suspenders/app_builder.rb

```diff
@@ -231,6 +231,14 @@ def configure_action_mailer_in_specs
       copy_file 'action_mailer.rb', 'spec/support/action_mailer.rb'
     end
 
+    def configure_helper_spec_helpers_in_specs
+      copy_file 'helper_spec_helpers.rb', 'spec/support/helper_spec_helpers.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.